src/library_async.js

```diff
@@ -40,6 +40,9 @@ addToLibrary({
     instrumentWasmImports(imports) {
 #if ASYNCIFY_DEBUG
       dbg('asyncify instrumenting imports');
+#endif
+#if ASSERTIONS && ASYNCIFY == 2
+      assert('Suspender' in WebAssembly, 'JSPI must be enabled.');
```

Comment:
How about "JSPI not support by current browser.  Perhaps it needs to be enabled via flags?"

Reply:
/s/browser/runtime? This may be node too. What do we usually refer to the runtime as?

Reply:
I guess it's environment in settings.

Reply:
Fair enough.